backend/llm_integration.py

```
@@ -360,10 +407,11 @@ def get_local_llm(model_path: str, tensor_split: List[float] = None) -> Any:
         logger.warning("llama_cpp not installed")
         return None
 
-        
-    if not model_path or not os.path.exists(model_path):
+    resolved = _resolve_model_path(model_path)
+    if not resolved or not os.path.exists(resolved):
         logger.info(f"Model not found at {model_path}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)